Light up bot/lightupbot.py

```diff
@@ -5,7 +5,7 @@
 from viam.rpc.dial import Credentials, DialOptions
 from viam.components.camera import Camera
```

Comment:
This line does not need to be here any more, since you are not getting or using images from it in this script.  I think not the following line as well.  Also not the PIL line?